Example/SwiftUIExample/MobilePaymentsSwiftUIExample/Screens/Home/HomeView.swift

```diff
@@ -0,0 +1,220 @@
+import SquareMobilePaymentsSDK
+import SwiftUI
+
+#if canImport(MockReaderUI)
+import MockReaderUI
+#endif
+
+struct HomeView: View {
+
+    @State private var presentingPermissionsView: Bool = false
+    @State var viewModel: HomeViewModel
+    
+    private let viewHolder: MobilePaymentsSDKViewHolder = MobilePaymentsSDKViewHolder()
+    private var mobilePaymentsSDK: SDKManager { viewModel.mobilePaymentsSDK }
+    private var isIPad: Bool {
+        UIDevice.current.userInterfaceIdiom == .pad
+    }
+
+    init(viewModel: HomeViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        // Some APIs of SquareMobilePaymentsSDK require a UIViewController,
+        // you can use a UIViewControllerRepresentable with an empty UIViewController
+        // and put it on the bottom of a ZStack
+        ZStack {
+            viewHolder
+            VStack {
+                headerView
+                contentView
+                Spacer()
+            }
+            .alert(isPresented: $viewModel.showPaymentStatusAlert) {
+                paymentStatusAlert
+            }
+        }
+        .padding([.leading, .trailing], isIPad ? 50 : nil)
+        .padding([.top, .bottom])
+        .background(Color.white)
```

Comment:
Is it intentional that this app doesn't support dark mode?

Reply:
Yeah per mary it's only light mode for now. Also set the User interface style to light for the app so that MPSDK screens are also forced light mode.